driver/pelican/main.go

```diff
@@ -25,6 +26,17 @@ type stateMsg struct {
 	Fan             *bool    `msgpack:"fan"`
 }
 
+// Object API Result Structs
```

Comment:
Remove these from `main.go`. All of the XML logic and any types for parsing XML should be fully encapsulated in `pelican.go`. The idea is that `main.go` doesn't care about how the Pelican API works, it just uses the data we get from the API and deals with Bosswave stuff.